build-a-react-native-application-for-ios-and-android-from-start-to-finish/notes/18-customize-the-stacknavigator-header-with-react-navigation-in-a-react-native-app.md

```diff
@@ -0,0 +1,32 @@
+# Customize the StackNavigator Header with React Navigation in a React Native App
+
+**[📹 Video](https://egghead.io/lessons/react-native-customize-the-stacknavigator-header-with-react-navigation-in-a-react-native-app)**
+
+## Configuration
+
+[Stack navigation](https://reactnavigation.org/docs/2.x/stack-navigator/) adds an automatic header, which contains the back button to go back to the previous screen. This can be turned off per component by adding `static navigationOptions = { header: null }`.
```

Comment:
```suggestion
🤔  [Stack navigation](https://reactnavigation.org/docs/2.x/stack-navigator/) adds an automatic header, which contains the back button to go back to the previous screen. This can be turned off per component by adding `static navigationOptions = { header: null }`.
```